pubspec.lock

```diff
@@ -209,6 +209,11 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.0"
+  flutter_localizations:
+    dependency: "direct main"
+    description: flutter
+    source: sdk
+    version: "0.0.0"
```

Comment:
@luongvo I think we only need two line for setup flutter_localizations as : 
flutter_localizations:
    sdk: flutter